java/org/contikios/cooja/GUI.java

```diff
@@ -1530,19 +1530,18 @@ public Boolean work() {
           list.addPopupMenuItem(null, true);
           tabbedPane.addTab("Contiki error", new JScrollPane(list));
         }
+        // Compilation output.
+        MessageListUI compilationOutput = null;
+        if (e instanceof MoteType.MoteTypeCreationException ex && ex.hasCompilationOutput()) {
+          compilationOutput = (MessageListUI) ex.getCompilationOutput();
+        } else if (e.getCause() instanceof MoteType.MoteTypeCreationException ex && ex.hasCompilationOutput()) {
```

Comment:
Will this not cause a `NullPointerException` if e is `null`?

Reply:
It does! Fixed.